packages/devextreme/js/ui/chat.d.ts

```diff
@@ -151,6 +160,20 @@ export type Message = {
     typing?: boolean;
 };
 
+/** @public */
+export type MessageTemplateData = {
+    readonly component: dxChat;
+    readonly message?: Message;
+    readonly text?: string;
```

Comment:
We don't need text property, it's included in `message`

Reply:
oh, right, fixed ty